client/core/trade.go

```diff
@@ -63,11 +59,9 @@ func (match *matchTracker) parts() (*order.UserMatch, *db.MatchMetaData, *db.Mat
 // trackedCancel is always associated with a trackedTrade.
 type trackedCancel struct {
 	order.CancelOrder
-	preImg  order.Preimage
-	matches struct {
-		maker *msgjson.Match
-		taker *msgjson.Match
-	}
```

Comment:
I can revert this if it's not an improvement, but for some reason it confused me more than having the `Match`es not in a struct.